### PR TITLE
Update test to not depend on space name

### PR DESCRIPTION
**Pull request by Roblinde**

### CODE REVIEW

File: tests/routes/test_settings.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def test_reset_restores_defaults(self):
 
         # Reset session
         response = self.app.post('/settings/reset').data
-        self.assertIn(b'The example app space v1', response)
+        self.assertIn(b'Loaded from local file', response)
```
Comment (floelhoeffel) [inline tests/routes/test_settings.py]: Loaded from local file?

Comment (Roblinde) [reply tests/routes/test_settings.py]: That's what the html says if you're loading the session locally

Comment (floelhoeffel) [reply tests/routes/test_settings.py]: Where do you change that test to evaluate that instead of the space name?

Comment (floelhoeffel) [reply tests/routes/test_settings.py]: ah I guess it evaluates the full string of that form?

Comment (Roblinde) [reply tests/routes/test_settings.py]: Yes, it reads the entire html response as a string and verifies some minor things.

**Pull request merged**